src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -203,8 +239,46 @@ const DriverOverviewCard: React.FC<DriverOverviewCardProps> = ({ data }) => {
                         COMPLETION OF DELIVERIES
                     </Text>
                 </View>
-                <View style={[styles.flexColumn, { width: "100%" }]}>{header}</View>
-                <View style={[styles.tableSection, styles.flexColumn]}>{table}</View>
+                {deliveries.length ? (
+                    <View style={styles.tableContainer}>
+                        <View
+                            style={[
+                                styles.h2text,
+                                styles.collectionOrDeliveryHeader,
+                                styles.flexRow,
+                            ]}
+                        >
+                            <Text style={styles.collectionOrDeliveryHeader}>Deliveries</Text>
+                            <FontAwesomeIconPdfComponent
+                                faIcon={faTruck}
+                            ></FontAwesomeIconPdfComponent>
+                        </View>
+                        <View style={[styles.flexColumn, { width: "100%" }]}>
+                            {deliveriesHeader}
+                        </View>
```

Comment:
You can pull this out as a separate component with headerName, icon, header and itemList as params (maybe more those are the differences that I see) and then reuse it

Reply:
Done, thank you